0.3/Arcs/source/Launcher.js

```diff
@@ -150,10 +161,14 @@ ${style}
       };
     }
     _onDelete(e) {
-      if (e.data.key > 0) {
-        this._views.get('arcs').remove(this._props.arcs[e.data.key]);
+      const arcId = e.data.key;
```

Comment:
Should we validate that we're not deleting an arc that we shouldn't here as well?

Reply:
Good suggestion. This change just relies on no UI affordance existing to enact the delete. Per my comment above I'm going to look at changing `New Arc` to not be an arc, and then we can just allow any arc to be deleted here.